the-way-we-work.md

```diff
@@ -0,0 +1,664 @@
+
+### 1.2 Why have we created The Way We Work?
+
+Recent surveys of NHSUK developers have told us that in order to do their best work they need a clearer explanation of their roles and more support from the leadership team. This document is our attempt to meet those needs.
+
+The leadership team have also been receiving strong messages from both the Director of Product Delivery, its Head of Engineering, and our own Head of Delivery about the urgent need to re-focus our activities to secure improvements in certain key areas. This document explains the systems which we believe will bring about those improvements.
+
+### 1.3 How we designed The Way We Work
+
+We believe in:
+
+- Growth: Everyone has the potential to improve
+- Autonomy: People work best when they control how they work
+- Mastery: People work best when they are always learning
+- Purpose: People work best when they choose what to work on
+- Support: People work best when they are supported by their colleagues
+- Structure: Progress is best made within an iterative and incremental system
+
+Therefore:
+
+- We will focus on future growth, not on past performance
+- We will ensure individuals choose what to work on and how to do that work
+- We will ensure individuals have the time to learn new things
+- We will ensure individuals understand their role and responsibilities
+- We will ensure individuals at all levels receive the support they need
+- We will expand the current quarterly objective setting cycle to accommodate the above
+
+### 1.4. How we will measure our success
+
+The Way We Work will be judged a success if it helps NHSUK software developers rapidly deliver value to users whilst improving the technical quality of our products.
+
+The leadership team will be held accountable for those business outcomes. The Way We Work is not intended to be a way to assess the performance of individuals.
+
+The leadership team has decided to measure their success in 3 broad categories:
+
+- Software Engineering Quality Dashboard
+- Individual Contribution
+- Team Participation
+
+**Software Engineering Quality Dashboard**
+
+The dashboard contains a number of technical measures for each of our products and we have been instructed to improve those scores.
+
+The leadership team will monitor progress towards improving our scores, which will be achieved by developers completing objectives as part of their 20% time.
+
+**Individual Contribution**
+
+Users are happy when value is delivered continuously and developers are happy when a piece of work is complete. 
+
+The leadership team will monitor how often developers are able to contribute code which is deployed to production.
+
+**Team Participation**
+
+We do our best work when we work together and when we help each other to improve.
+
+The leadership team will monitor activity on Jira tickets.
+
+TODO: Will we also monitor how many pair programming (or mentoring) sessions are taking place?
```

Comment:
One way of monitoring this is for the developers to comment on the Jira tickets who have they paired with to accomplish a particular task.

Reply:
I will add this in to section 3.3